README.md

```diff
@@ -1,3 +1,155 @@
-# Template Repository
+# Bitwarden Workflow Linter
```

Comment:
❓ Did you happen to `npm ci` before these commits? I want to ensure Prettier runs where appropriate. Additionally, you can plug any other linters in there.

Reply:
I had not run `npm ci`. I have now. This being said, I haven't seemed to be able to find a way to integrate `black` into prettier

(Note to self: we'll need to backport `bitwarden/template` into `bitwarden/gh-actions`)

Reply:
Ah, I forgot I was also using `pylint`. I'll see if I can find a way of integrating that with Prettier

Reply:
Instead of integrating directly with `prettier`, I set both `pylint` and `black` to run on `*.py` files